packages/opentelemetry-exporter/src/index.js

```diff
@@ -0,0 +1,288 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2021
+ */
+
+'use strict';
+
+const { backendConnector: instanaBackendConnector, environment: instanaEnvironment } = require('@instana/serverless');
+const { ExportResultCode } = require('@opentelemetry/core');
+const instanaEndpointUrlEnvVar = 'INSTANA_ENDPOINT_URL';
+const instanaAgentKeyEnvVar = 'INSTANA_AGENT_KEY';
+
+/** @typedef {import('@opentelemetry/sdk-trace-base').ReadableSpan} ReadableSpan */
+/** @typedef {import('@instana/core/src/tracing/cls').InstanaBaseSpan} InstanaBaseSpan */
+
+// The exporter is based on the parser from the Agent:
+// eslint-disable-next-line max-len
+// https://github.com/instana/agent/blob/51da097e99e9d05ac5c0b5b7dd9275a12c20685d/agent-optional-parent/agent-open-telemetry/src/main/java/com/instana/agent/main/impl/otel/OpenTelemetryTraceEndpoint.java#L69
```

Comment:
_thought:_ This is a link to a private repo from our public code base. There is nothing wrong with that in principle, but it feels a bit odd to me, just saying.

Reply:
Good catch. I will pay more attention before adding links to the code.
We should not have private repo links in the a public repo.